bundle/jsky.app.ot/src/main/scala/edu/gemini/catalog/ui/QueryResultsFrame.scala

```diff
@@ -69,12 +72,27 @@ object QueryResultsFrame extends Frame with PreferredSizeFrame {
     }
   }
 
+  private object InFOVRenderer {
+    val inFovIcon: ImageIcon = Resources.getIcon("bullet/bullet_green.png")
+    val outFovIcon: ImageIcon = Resources.getIcon("bullet/bullet_red.png")
+    val icons: Map[GuideInFOV, ImageIcon] = Map(InsideFOV -> inFovIcon, OutsideFOV -> outFovIcon)
```

Comment:
Any reason to not have a [total] method here? Do we ever do anything other than `.get`?